features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3107,6 +3107,9 @@
     <!-- Configuration to enable or disable claim filtering based on scope. -->
     <EnableScopeBasedClaimFiltering>{{authentication.enable_scope_based_claim_filtering}}</EnableScopeBasedClaimFiltering>
 
+    <!-- Configuration to strictly restrict claim filtering only as requested by SP. -->
+    <RestrictSPRequestedClaimFiltering>{{authentication.restrict_sp_requested_claim_filtering}}</RestrictSPRequestedClaimFiltering>
```

Comment:
shall we rename it as `allow_request_claims_only`?

Reply:
Renamed to `allow_sp_requested_fed_claims_only`